test/api_volume_inspect_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/alibaba/pouch/test/request"
+
+	"github.com/go-check/check"
+)
+
+// APIVolumeInspectSuite is the test suite for volume inspect API.
+type APIVolumeInspectSuite struct{}
+
+func init() {
+	check.Suite(&APIVolumeInspectSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *APIVolumeInspectSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+}
+
+// TestVolumeInspectOk tests if inspecting a volume is OK.
+func (suite *APIVolumeInspectSuite) TestVolumeInspectOk(c *check.C) {
+	//Here creates a volume with the name "TestVolume".
+	vol := "TestVolume"
+	obj := map[string]interface{}{
+		"Driver": "local",
+		"Name":   vol,
+	}
+	path := "/volumes/create"
+	body := request.WithJSONBody(obj)
+	request.Post(path, body)
```

Comment:
Please make request.Post return error and do assert here, since this may error here but we ignore this.